assets/i18n/en.json

```diff
@@ -113,6 +113,9 @@
     "I18N_FORMS_TYPE_NUMBER_AT_LEAST": "Please enter a number that is at least <[minValue]>.",
     "I18N_FORMS_TYPE_NUMBER_AT_MOST": "Please enter a number that is at most <[maxValue]>.",
     "I18N_FORMS_TYPE_NUMBER_INVALID_DECIMAL": "Please enter a valid decimal number.",
+    "I18N_FRACTIONS_DESCRIPTION": "Fractions represent numbers \"between\" the counting numbers. For example, if I cut a cake into four equal pieces, and eat one of them (yum!), I need a number between 0 and 1 to represent how much I've eaten. <a href=\"/explore/ereONiVLp-VE\">Start the first lesson</a> to learn more.",
```

Comment:
OK, on second thought -- I really think that we shouldn't translate these at all. Here are the reasons:
- The current landing pages are a test. We don't know if we would keep any or all of them in the long run.
- The lessons are written in English only, so it makes sense for the landing page to be in English. It would be a strange transition to go from a non-English landing page to an English lesson, IMO.
- Given the above, I don't want the folks on translatewiki to spend time translating these strings if we aren't going to use them in the long run.

Would you agree? If so, can we remove the translations? Sorry about this.